packages/Confirmation/src/components/TriggerButton/TriggerButton.js

```diff
@@ -6,15 +6,17 @@ const propTypes = {
   children: PropTypes.node,
   confirmId: PropTypes.string,
   icon: PropTypes.node,
-  isConfirmOpen: PropTypes.bool.isRequired,
-  onOpenConfirm: PropTypes.func.isRequired,
+  isConfirmOpen: PropTypes.bool,
+  onOpenConfirm: PropTypes.func,
```

Comment:
Why can't these remain `isRequired`?

Reply:
🤔   Previously I was getting missing prop type complaints when these were `required` when i was cloning these props onto the child - https://github.com/acl-services/paprika/blob/xd-7194-simplify-confirmation-component-api/packages/Confirmation/src/Confirmation.js#L96 and was a large part the reason i originally exposed these props to the consumer. However I don't see it complaining now so I have readded.

Reply:
Actually re-ran `yarn` and i still see the proptype errors when running unit tests and in storyboards, so setting back to not required.
![image](https://user-images.githubusercontent.com/4040102/71117565-39ac8600-218b-11ea-8794-361e13aaedb2.png)

Reply:
Perhaps another reason to use context over React cloning when possible.